config/endorsed/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.sakaiproject</groupId>
+        <artifactId>master</artifactId>
+        <version>12-SNAPSHOT</version>
+        <relativePath>../master/pom.xml</relativePath>

```

Comment:
You're right, sorry for the typo. I don't understand why my build doesn't fail... I'll fix it.